openstack/check.py

```diff
@@ -779,14 +796,11 @@ def check(self, instance):
             project = self.get_scoped_project(instance)
 
             # Restrict monitoring to non-excluded servers
-            excluded_server_ids = self.init_config.get("exclude_server_ids", [])
-            servers = list(
-                set(self.get_servers_managed_by_hypervisor()) - set(excluded_server_ids)
-            )
+            server_ids = self.get_servers_managed_by_hypervisor(instance)
```

Comment:
Yeah, let's do this to ensure there are no changes in behavior. I updated the docs.